version-check

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```

Comment:
Could you put this script in `utils/scripts`, and prepare a CLI that will supports all tracers (even if only java is supported).

As now, I see : 

```bash
./utils/script/version-check.sh <tracer_name> <SCENARIO_NAME> [-w WEBLOG_VARIANT]
```

probably a `-n` option to give how versions it should check ?